cmake/FindREADLINE.cmake

```diff
@@ -0,0 +1,47 @@
+# - Try to find readline include dirs and libraries 
```

Comment:
I'm not quite happy with this find module as it is exporting variables and such. Modern find modules should export targets with all needed variables (see https://pabloariasal.github.io/2018/02/19/its-time-to-do-cmake-right/). I have a branch where I started modernizing existing find modules.

To avoid being a show-stopper I'm okay with this form for now, I just wanted to create a bit sensitivity for that topic :wink:

Reply:
Good to know!